scalanet/src/io/iohk/scalanet/peergroup/InetPeerGroupUtils.scala

```diff
@@ -1,29 +1,47 @@
 package io.iohk.scalanet.peergroup
 
 import java.net.{InetSocketAddress, ServerSocket}
-
-import io.netty.util
+import io.netty
+import io.netty.util.concurrent.{Future, GenericFutureListener}
 import monix.eval.Task
 
+import java.util.concurrent.CancellationException
+
 object InetPeerGroupUtils {
```

Comment:
This class is getting ever more unrelated utility methods, some of which should be moved to tests (`aRandomNode`) and now most of which are about netty futures. Would be good to clean it up a bit, make it private to scalanet.